style.css

```diff
@@ -118,4 +118,340 @@ div.contact ul a {
   padding-right: 20px;
 }
 
-/* section end */
+/* section explain end */
+
+/* section works start */
+.works {
+  background: rgb(247, 245, 245);
+  margin: 0%;
+  padding: 0%;
+  width: 378px;
+  position: relative;
+  height: 3500px;
+  margin-top: 90px;
+}
```

Comment:
- [ ] According to the project requirement, the main container of the projects has a display: grid. please implement it